### ISSUE TITLE
Possible to overwrite a particular translation?

### ISSUE THREAD

**Issue by christhesoul** [onomojo/i18n-country-translations#24]
Hey all. Thanks for your efforts in making this gem.

I'm working with a client who in turn works with Russian customers. They are sensitive about it being referred to as "Russian Federation" and not simply "Russia" in English.

Is there a way I can overwrite translations in my own .yml file to cope with such sensitivities?

Or an alternative solution?

Thanks again!
Chris

**Comment by onomojo** [comment]
You could just submit a pull request to change the name to Russia.

**Comment by neohunter** [comment]
Just checked, you don't need to submit a yml, simply put on a yml file on `config/locale/countries.yml`

```
en:
  countries:
    RU: "Russian Federation"

**Comment by onomojo** [comment]
Yes! You can override any translation by defining your own YAML file with the same keys. Rails i18n merges translation files, and files loaded later take precedence. Simply add your own `config/locales/countries.en.yml` with:

```yaml
en:
  countries:
    RU: Russia
```

This will override just the entries you specify. Closing as this is a usage question rather than a bug.
